### PR TITLE
fix #181 display full database, update select slot UG

### LINKED ISSUE #181: [PE-D][Tester E] Search command displays all modules with "search /title /"
All modules stored in the database are displayed when `search /title /`  is issued. 

-------------
Labels: `severity.VeryLow` `type.FeatureFlaw`
original: redders7/ped#8

### PR DESCRIPTION
**Pull request by amitrahman1026**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -487,7 +487,20 @@ Selects a timeslot to be added to the user timetable.
 Format: `select /module MODULE_CODE /type LESSON_TYPE /code CLASS_NO`
 
 * The `CLASS_NO` will be a number.
-* The `LESSON_TYPE` can be either *lecture*, *tutorial*.
+* The `LESSON_TYPE` can be any of the following:  
+  *  TUTORIAL                   represented by inputs starting with `TUT`
```
Comment (owenl131) [inline docs/UserGuide.md:491]: Don't tell them how we are parsing it lol. Just tell them what is accepted. I suggest just give them two options - spell out in full or short form.

File: src/main/java/seedu/duke/parser/LessonTypeParser.java [hunk 2]
```diff
@@ -22,7 +22,7 @@ public static LessonType parse(String input) throws IllegalArgumentException {
         if (input.equals("plec") || (input.startsWith("pac") && input.contains("lec"))) {
             return LessonType.PACKAGED_LECTURE;
         }
-        if (input.equals("plec") || (input.startsWith("pac") && input.contains("tut"))) {
+        if (input.equals("ptut") || (input.startsWith("pac") && input.contains("tut"))) {
```
Comment (owenl131) [inline src/main/java/seedu/duke/parser/LessonTypeParser.java:25]: nice

**Review by owenl131 — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/duke/command/SearchModuleCommand.java [hunk 3]
```diff
@@ -91,6 +98,15 @@ private void processParams() throws YamomException {
         toSearchLevel = processSearchLevel(level);
         // validate semester
         toSearchSemester = processSemester(semester);
+
```
Comment (owenl131) [inline src/main/java/seedu/duke/command/SearchModuleCommand.java:101]: I think we are just checking that if the moduleCode is not null then it shouldn't be empty right. In which case no need to && check with moduleTitle.

Comment (owenl131) [reply src/main/java/seedu/duke/command/SearchModuleCommand.java:101]: Also, can use "".equals(toSearchModuleCode) instead of Objects.equals

Comment (amitrahman1026) [reply src/main/java/seedu/duke/command/SearchModuleCommand.java:101]: Will fix

Comment (amitrahman1026) [reply src/main/java/seedu/duke/command/SearchModuleCommand.java:101]: > I think we are just checking that if the moduleCode is not null then it shouldn't be empty right. In which case no need to && check with moduleTitle.

good catch

**Review by owenl131 — APPROVED** [review]

**Pull request merged**